bot/exts/info/reddit.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 import logging
+import html
```

Comment:
We don’t really need the full library here. 
```suggestion
from html import unescape
```